ci/update-certificates.sh

```diff
@@ -90,6 +94,32 @@ for row in $(echo $mapping | jq -c '.[]'); do
 
 done
 
+# store the CA cert in env secrets file
+spruce json secrets-updated/secrets.yml \
+  | jq --arg ca "$(spruce json secrets-in-master/secrets.yml | jq -r '.secrets.ca_cert')" ".secrets.ca_cert = \$ca" \
```

Comment:
Need to rework how we deal with rotating CA:
1. Add new CA to master secrets
1. Add new CA to (tooling,development,staging,production) secrets
1. Deploy all bosh environments (new CA trusted)
1. Generate new master-bosh secrets & deploy master-bosh
1. Generate new deployment-bosh secrets & deploy each
1. Remove old CA from master secrets
1. Remove old CA from (tooling,development,staging,production) secrets
1. Deploy all bosh environments (old CA untrusted)

It might make sense after all to use a `secrets-common.yml` after all for CA/pubkey and an additional store for master-bosh which will _only_ contain secrets for that deployment.

Reply:
Sounds like a good idea.